pytximport/core/_tximport.py

```diff
@@ -553,7 +576,17 @@ def tximport(
 
         log(25, f"Saving the gene-level expression to: {output_path}.")
 
-        if output_format == "h5ad":
+        if output_format.lower() == "summarizedexperiment":
+            from summarizedexperiment import SummarizedExperiment
```

Comment:
Duplicate import, already imported above. Should be simplified to only import in one place.

Reply:
My understanding of the code is output_type and output_format seem to be separated because of various combinations between them. If the type is not SE but the format is SE then the packages are not loaded.

Reply:
nvm, the recent commit should take care of this.